projects/plugins/wpcomsh/wpcomsh.php

```diff
@@ -132,6 +132,7 @@
 require_once __DIR__ . '/notices/media-library-private-site-cdn-notice.php';
 require_once __DIR__ . '/notices/anyone-can-register-notice.php';
 require_once __DIR__ . '/notices/feature-moved-to-jetpack-notices.php';
+require_once __DIR__ . '/notices/manage-elementor-beta-notice.php';
```

Comment:
Rather than a single-purpose file, I would put this in https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/wpcomsh/plugin-hotfixes.php.